modules/oxide-components/tsconfig.app.json

```diff
@@ -7,7 +7,8 @@
     "lib": [
       "ES2020",
       "DOM",
-      "DOM.Iterable"
+      "DOM.Iterable",
+      "ES2022.Intl"
```

Comment:
You can upgrade the whole lib from es2020 to es2022 if you need to. TinyMCE already does.
https://github.com/tinymce/tinymce/blob/867d1ad31ef6e9d874f937334efac2be4bb2b10f/tsconfig.shared.json#L6